CHANGELOG.md

```diff
@@ -2,7 +2,16 @@
 
 ## Redpanda Chart
 
-### [Unreleased](https://github.com/redpanda-data/helm-charts/releases/tag/redpanda-FILLMEIN) - YYYY-MM-DD
+### [Unreleased](https://github.com/redpanda-data/helm-charts/releases/tag/redpanda-5.8.13) - YYYY-MM-DD
```

Comment:
NIT:

Why link of Unreleased points to `https://github.com/redpanda-data/helm-charts/releases/tag/redpanda-5.8.13`?

Reply:
Whoops, that's a mistake on my behalf! I'll fix it the next time I update the notes.